src/sdk/pynni/nni/compression/torch/pruning/amc/lib/net_measure.py

```diff
@@ -85,11 +85,11 @@ def measure_layer(layer, x):
     return
 
 
-def measure_model(model, H, W):
+def measure_model(model, H, W, device):
     global count_ops, count_params
     count_ops = 0
     count_params = 0
-    data = torch.zeros(2, 3, H, W).cuda()
+    data = torch.zeros(2, 3, H, W).to(device)
```

Comment:
The input channel is limited to 3 which may limit the usage scenario of AMC pruner (only suitable for the models whose input channel equals 3)? 
By the way, `measure_model` looks duplicated with `count_flops_params` in `nni/compression/torch/utils/counter.py`, shall we reuse `count_flops_params` here?

Reply:
Thanks for good catch, AMC uses 2 flops counters,  one for each layer and another for entire model, but count_flops_params currently can not  count flops for each layer.  So we can only replace one counter with `count_flops_params`.   We are planning to refactor the flops counters in AMC in another PR.

Reply:
Got it~ Thanks~